lib/Core/Executor.cpp

```diff
@@ -1533,7 +1544,11 @@ void Executor::executeInstruction(ExecutionState &state, KInstruction *ki) {
         ref<Expr> value = evalConstant(si->getCaseValue(i));
 #endif
         ref<Expr> match = EqExpr::create(cond, value);
-        isDefault = AndExpr::create(isDefault, Expr::createIsZero(match));
+
+        // default value cannot contain this target
+        defaultValue = AndExpr::create(defaultValue, Expr::createIsZero(match));
```

Comment:
@delcypher The point of this comment is to say that if this case is the target, then the associated value won't be part of the default target.